bin/hdfcoinc/pycbc_multiifo_dtphase

```diff
@@ -0,0 +1,169 @@
+#!/bin/env python
+""" Create a file containing the phase and amplitude, 
+correlations between two detectors by
+doing a simple monte-carlo
```

Comment:
-> 'correlations in phase difference, time difference and relative amplitude between two or more detectors'